Cargo.toml

```diff
@@ -65,6 +65,7 @@ rustls = { version = "0.21", features = ["dangerous_configuration"], optional =
 zeroize = { version = "1.6", features = ["zeroize_derive"] }
 tokio = { version = "1.32", features = ["time", "rt"], optional = true }
 pcsc = { version = "2.8", optional = true }
+async-recursion = "1.0.5"
```

Comment:
Nit: I don’t think it’s worth to depend on a new proc-macro crate for so little, but it’s okay to not address this right now